client/src/modules/debtors/groups.update.html

```diff
@@ -23,9 +23,19 @@
 
           <div
             class="form-group has-feedback"
-            ng-class="{'has-error' : debtorGroup.color.$invalid && debtorGroup.$submitted}">
+            ng-class="{'has-error' : !GroupUpdateCtrl.group.color && debtorGroup.$submitted}">
             <label class="control-label" translate>FORM.LABELS.COLOR</label>
-            <input name="color" id="title-color" type="color" ng-model="GroupUpdateCtrl.group.color" class="form-control" ng-value="GroupUpdateCtrl.group.color">
+
+            <ui-select tagging ng-model="GroupUpdateCtrl.group.color" theme="bootstrap">
+              <ui-select-match placeholder="Pick one...">
+                  <i class="fa fa-circle" style="color: {{$select.selected.value}}; font-sze:14px"></i>&nbsp;
+                   <span translate>{{$select.selected.name}}</span>
+              </ui-select-match>
+              <ui-select-choices repeat="val.value as val in GroupUpdateCtrl.colors | filter: {'name' :$select.search}">
+                  <div ng-click="GroupUpdateCtrl.group.color = val.value">
```

Comment:
Do this `ngClick` need to be defined?  You already set the `ng-model` to this value.

The proper way to get a selection from a `uiSelect` is using the `onSelect` callback anyway.  [Here is the documentation](https://github.com/angular-ui/ui-select/wiki/ui-select#events).